packages/patterns-react/.storybook/webpack.config.js

```diff
@@ -70,7 +70,6 @@ const styleLoaders = [
       data: `
         $feature-flags: (
           ui-shell: true,
-          enable-css-custom-properties: true
```

Comment:
Actually, I did a a separate PR to have it turned on only if the carbon expressive theme is turned on:
https://github.com/carbon-design-system/ibm-dotcom-library/pull/787

Reply:
kk - putting it back in there

Reply:
kk - putting it back in there